target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -158,6 +158,13 @@ engenius,ens202ext-v1)
 	ucidef_set_led_rssi "rssimedium" "RSSIMEDIUM" "$boardname:amber:rssimedium" "wlan0" "33" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "$boardname:green:rssihigh" "wlan0" "67" "100"
 	;;
+engenius,enstationac-v1)
+	ucidef_set_led_wlan "wlan" "WLAN" "$boardname:green:wlan" "phy0tpt"
```

Comment:
You have a trigger in device tree, so this line can just be dropped.

Reply:
this makes it the same as ens202ext so I merged them